app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsFragmentTest.kt

```diff
@@ -79,29 +111,40 @@ import org.oppia.android.util.parser.html.HtmlParserEntityTypeModule
 import org.oppia.android.util.parser.image.GlideImageLoaderModule
 import org.oppia.android.util.parser.image.ImageParsingModule
 import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
 import javax.inject.Inject
 import javax.inject.Singleton
 
-/** Tests for [AdministratorControlsFragment]. */
 @RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
 @Config(
   application = AdministratorControlsFragmentTest.TestApplication::class,
   qualifiers = "port-xxhdpi"
 )
 class AdministratorControlsFragmentTest {
+
   @get:Rule
   val initializeDefaultLocaleRule = InitializeDefaultLocaleRule()
 
+  @get:Rule
+  val accessibilityTestRule = OppiaTestRule()
```

Comment:
`accessibilityTestRule ` to `oppiaTestRule`

Reply:
done